src/main/resources/mappings/opendistro-ism-config.json

```diff
@@ -442,6 +442,20 @@
               }
             }
           }
+        },
+        "ism_template": {
```

Comment:
Do we need to increment the schema of the mappings or has it already been incremented after the last release?

Reply:
good point, I think we can increase the version to 7 in the security PR.